src/shared/sentry/sentryadapter.cpp

```diff
@@ -37,17 +41,25 @@ SentryAdapter::SentryAdapter() { MZ_COUNT_CTOR(SentryAdapter); }
 SentryAdapter::~SentryAdapter() { MZ_COUNT_DTOR(SentryAdapter); }
 
 void SentryAdapter::init() {
+  if (m_initialized) {
+    return;
+  }
   if (!Feature::get(Feature::Feature_sentry)->isSupported()) {
     return;
   }
-  if (QString(Constants::SENTRY_DSN_ENDPOINT).isEmpty() ||
-      QString(Constants::SENTRY_ENVELOPE_INGESTION).isEmpty()) {
+  auto settings = SettingsHolder::instance();
+  if (!settings->hasSentryEndpoint() || !settings->hasSentryDSN()) {
+    // If we have no info on where to put crash info, let's
+    // query that and retry later.
     logger.error() << "Sentry failed to init, no sentry config present";
+    auto t = new TaskSentryConfig();
```

Comment:
In this instance it's not as prevalent but generally speaking I try to avoid one character var names
```suggestion
    auto sentryConfig = new TaskSentryConfig();
```